pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -990,6 +1011,53 @@ def get_channel_contributions_forward_pass_grid(
             },
         )
 
+    def plot_channel_parameter(self, param_name: str, **plt_kwargs: Any) -> plt.Figure:
```

Comment:
Were did this method move from. If the class changed then this might explain the plotting tests failing